packages/kbn-grid-layout/grid/grid_panel/grid_panel.tsx

```diff
@@ -34,6 +37,37 @@ export const GridPanel = forwardRef<HTMLDivElement, GridPanelProps>(
     { panelId, rowIndex, renderPanelContents, interactionStart, gridLayoutStateManager },
     panelRef
   ) => {
+    const [dragHandleApi, setDragHandleApi] = useState<DragHandleApi | null>(null);
+
+    useEffect(() => {
+      const onDropEventHandler = (dropEvent: MouseEvent) => interactionStart('drop', dropEvent);
+      /**
+       * Subscription to add a singular "drop" event handler whenever an interaction starts -
+       * this is handled in a subscription so that it is not lost when the component gets remounted
+       * (which happens when a panel gets dragged from one grid row to another)
+       */
+      const dropEventSubscription = gridLayoutStateManager.interactionEvent$.subscribe((event) => {
+        if (!event || event.id !== panelId) return;
+
+        /**
+         * By adding the "drop" event listener to the document rather than the drag/resize event handler,
+         * we prevent the element from getting "stuck" in an interaction; however, we only attach this event
+         * listener **when the drag/resize event starts**, and it only executes once (i.e. it removes itself
+         * once it executes, so we don't have to manually remove it outside of the unmount condition)
+         */
+        document.addEventListener('mouseup', onDropEventHandler, {
+          once: true,
+          passive: true,
```

Comment:
Making these event listeners passive means that we do not have to call `preventDefault` - see https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#passive.